frontend/src/modules/member/pages/member-merge-suggestions-page.vue

```diff
@@ -14,27 +14,40 @@
       Here you can check all the merging suggestions.
     </div>
 
-    <div v-if="loading || count > 0" class="panel">
+    <div v-if="loading || count > 0" class="panel !p-0">
       <!-- Header -->
-      <header class="flex items-center justify-between pb-4">
-        <button
-          type="button"
-          class="btn btn--transparent btn--md"
-          :disabled="loading || offset <= 0"
-          @click="fetch(offset - 1)"
-        >
-          <span class="ri-arrow-left-s-line text-lg mr-2" />
-          <span>Previous</span>
-        </button>
-        <app-loading v-if="loading" height="16px" width="131px" radius="3px" />
-        <div
-          v-else
-          class="text-sm leading-5 text-gray-500 flex flex-wrap justify-center"
-        >
-          <div>{{ offset + 1 }} of {{ Math.ceil(count) }} suggestions</div>
+      <header class="flex items-center justify-between px-6 py-5 border-b">
+        <div class="flex items-center">
```

Comment:
Can you add a gap here?
Buttons don't have space between them
![Screenshot 2024-01-05 at 11 51 34](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/e3b7f314-48de-419b-8f71-82015911480e)
![Screenshot 2024-01-05 at 11 51 32](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/5877b96e-7c54-4a9f-9e26-3de03e9fcdfe)